lib/challenge.js

```diff
@@ -48,9 +62,15 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet = {
 
 // Answer Starts Here
+petName: "something",
+species: "somespecies",
+breed: "someBreed",
+petSound: [ {noise}],
```

Comment:
The value assigned to `some_pet.petSound` is syntactically incorrect. A method for making a noise is also missing.